src/index.js

```diff
@@ -20,15 +20,97 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+const initNum = 0;
+const digits = []; // 사용자가 입력하는 숫자를 자릿수별로 배열에 담는다. 예. 123 입력 -> [1, 2, 3]
+const userInputs = []; // 사용자가 입력한 숫자, 연산자 모음
+
+function calculate() {
+  const opr = userInputs[1];
+  if (opr === '+') {
+    return userInputs[0] + userInputs[2];
+  }
+  if (opr === '-') {
+    return userInputs[0] - userInputs[2];
+  }
+  if (opr === '*') {
+    return userInputs[0] * userInputs[2];
+  }
+  if (opr === '/') {
+    return userInputs[0] / userInputs[2];
+  }
+  return 0;
+}
+
+function render(visibleNum) {
+  function handleClickNumber(num) {
+    digits.push(num);
+    render(parseInt(digits.join(''), 10));
```

Comment:
```suggestion
    const number = parseInt(digits.join(''), 10); 
    render(number);
```
비교했을 때 어떤게 코드 가독성이 더 좋은 것 같나요?